serverlessworkflow/modules/ROOT/pages/persistence/postgresql-flyway-migration.adoc

```diff
@@ -0,0 +1,97 @@
+= Migrating Your PostgreSQL Database with a Kogito upgrade
+:flyway_install_url: https://flywaydb.org/download/community
+:flyway_migrate_existing_url: https://flywaydb.org/documentation/learnmore/existing
+
+Kogito provides an out-of-the-box way to manage your database schema changes with each upgrade. It uses Flyway to achieve this functionality.
+
+When you upgrade your Kogito version, by default it will pick up the latest changes available from Kogito and apply them automatically.
+
+== How to migrate
+
+=== Migrate using Flyway Config
+
+* Kogito provides a default mechanism for migrating your database while updating the Kogito version using the following `Flyway` properties (default value is `false`):
++
+[tabs]
+====
+Quarkus::
++
+--
+`quarkus.flyway.migrate-at-start=true`
+--
+Spring Boot::
```

Comment:
@akumar074 SB should probably not be mentioned here as it is not supported for SW